### PR TITLE
Fix Adoc header

### PR DESCRIPTION
**Pull request by agoncal**
https://docs.asciidoctor.org/asciidoc/latest/document/header/

> The document header may not contain empty lines. The first empty line the processor encounters after the document header begins marks the end of the document header and the start of the document body. 

https://docs.asciidoctor.org/asciidoc/latest/document/subtitle/

> The HTML 5 converter does not currently split the subtitle out from the document title when generating HTML from AsciiDoc. The document title is only partitioned into a main and subtitle in the output of the DocBook, EPUB 3, and PDF converters.

### CODE REVIEW

**Comment by holly-cummins** [comment]
This is super-nitpicky, but I think I preferred the old layout, with the logo first. 

Before: 

image

After:
image

@agoncal, I assume you think it's better this way? I'm happy to be persuaded since it's the opposite of important. :)

**Comment by agoncal** [comment]
Oh no @holly-cummins, it's not about aesthetics ;o) I realised that our Asciidoc header does not work. The Asiidoc rules for [the header are very strict](https://docs.asciidoctor.org/asciidoc/latest/document/header/). As soon as you have one empty line, that's the end of the header, you need to have the subtitle in a specific format (even though it does not render in HTML, but yes in PDF and EPUB), the writer names just underneath the title.... and no image in the header. 

So, basically, our header (variable definition, properties...) stopped working from line number 2  :oD

**Comment by agoncal** [comment]
@holly-cummins but what we can do is use `:front-cover-image: ` instead. I'm having a look how it shows in HTML and PDF...

**Comment by agoncal** [comment]
@holly-cummins I've added `:front-cover-image: image:quarkus-logo.png[]` which renders very nicely in PDF... but not in HTML
